scripts/movieReviews.js

```diff
@@ -1,21 +1,27 @@
-var localMovies = [
+var localData = [
```

Comment:
A more semantic/explanatory variable name might be helpful here (localMovies was more descriptive).